concepts/numbers/introduction.md

```diff
@@ -2,14 +2,12 @@
 
 There are two different types of numbers in JavaScript:
 
-- `number`: a numeric data type in the double-precision 64-bit floating point
-  format (IEEE 754). Examples are `-6`, `-2.4`, `0`, `0.1`, `1`, `3.14`,
-  `16.984025`, `25`, `976`, `1024.0` and `500000`.
-- `bigint`: a numeric data type that can represent _integers_ in the arbitrary
-  precision format. Examples are `-12n`, `0n`, `4n`, and `9007199254740991n`.
+- `number`: a numeric data type in the double-precision 64-bit floating point format (IEEE 754).
+  Examples are `-6`, `-2.4`, `0`, `0.1`, `1`, `3.14`, `16.984025`, `25`, `976`, `1024.0` and `500000`.
+- `bigint`: a numeric data type that can represent _integers_ in the arbitrary precision format.
+  Examples are `-12n`, `0n`, `4n`, and `9007199254740991n`.
 
-In contrast, in many other programming languages different numeric types can exist,
-for example: Integers, Floats, Doubles, or Bignums.
+In contrast, in many other programming languages different numeric types can exist, for example: Integers, Floats, Doubles, or Bignums.
```

Comment:
I mostly wanted to point out that other languages have many types, JS only has 1* (and bigint)

Reply:
That makes sense for sure, maybe it was just the placement that seemed to make the flow of paragraph more abrupt.  Maybe an into sentence mentioning this would flow differently?

`Many programming languages have specific numeric types to represent different types of numbers, but JavaScript only has two:`